tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -35,6 +37,23 @@ def run_optimize(args: Tuple[str, str]) -> None:
     study.optimize(objective, n_trials=20)
 
 
+def run_optimize_with_progbar(args: Tuple[str, str, int]) -> str:
```

Comment:
`run_optimize_with_progbar` returns progress bar as `str`, so how about this?

```suggestion
def get_completed_progbar(args: Tuple[str, str, int]) -> str:
```

If there exist a better name, please let me know!

Reply:
The functionality of the ```run_optimize_with_progbar``` function is not only to return the progress bar, but also to run optimize. In fact, not only the progress bar but also the generated trials are checked subsequently. Therefore, the ```run_optimize_with_progbar``` function is the ```run_optimize``` function plus the ability to return the progress bar, and I think the name is appropriate.

Reply:
I respectfully disagree with you. This function aims to get a completed progress bar that can be obtained by only using `optimize` method in Optuna. Since similar method `run_optimize` does return `None`, we expect `run_optimize_with_progbar` does also the same thing with progress bar option; we expect this returns `None`. Therefore the current name can mislead. 

cc: @toshihikoyanase

Reply:
In terms of the test of optimisation results, I'm not sure this test is really necessary because the same test without a progress bar is done on another test.

Reply:
Ok, renamed to ```get_completed_progbar``` and removed check of trials.